docs/01-installation.md

```diff
@@ -0,0 +1,61 @@
+---
+title: Installing Polynote
+layout: docs
+---
+
+# Installing Polynote
+
+> ## Warning!
+> Polynote is a web-based programming notebook tool. Like other notebook tools, a large part of its usefulness relies on
+> **arbitrary remote code execution.**
+>
+> It currently contains **no built-in security or authentication** of its own, and relies entirely on the user deploying
+> and configuring it in a secure way.
+> 
+> Polynote should only be deployed on a secure server which has its own security and authentication mechanisms which
+> prevent all unauthorized network access.
+>
+> You are solely responsible for any damage or other loss that occurs as a result of running Polynote.
+{:.warning} 
+
+## Download
+Polynote consists of a JVM-based server application, which serves a web-based client. To try it locally, find the latest
+release on the [releases page](https://github.com/polynote/polynote/releases){:target="_blank"} and download the attached
+`polynote-dist.tar.gz` file. Unpack the archive:
+
+```
+tar -zxvpf polynote-dist.tar.gz
+cd polynote
+```
+
+## Prerequisites
+- Polynote is currently only tested on Linux and MacOS, using the Chrome browser as a client. We hope to be testing
+  Windows (and other browsers) soon. Feel free to try it on your platform, and be sure to let us know about any issues
```

Comment:
Windows?

Reply:
I'll just edit it to say "other platforms and browsers" so as not to over-commit 😆

Reply:
:)

I think Windows support for clients (say, Chrome on Windows) is something we should probably look into. 

Running the actual server on Windows seems more daunting...